### PR TITLE
first push to my fork JoaquinGonzales

### PR DESCRIPTION
**Pull request by JoaquinGonzales**

### CODE REVIEW

File: config/config.json [hunk 1]
```diff
@@ -10,12 +10,13 @@
...
-		"resourceConfig" : "/config/resource.json",
-		"meetingsAPI" : "/lib/meetingsAPI",
-		"util" : "/util/util"
+		"tokenAPI" : "../../lib/tokenAPI",
+		"resourcesAPI" : "../../lib/resourcesAPI",
+		"endPoints" : "../../config/endPoints.json",
+		"resourceConfig" : "../../config/resource.json",
+		"servicesAPI" : "../../lib/servicesAPI",
+		"meetingsAPI" : "../../lib/meetingsAPI",
+		"util" : "../util/util"
```
Comment (AlejandroJeanCarlo) [inline config/config.json]: this should be like "/util/util" without .. 
as we agreed

File: lib/servicesAPI.js [hunk 2]
```diff
@@ -0,0 +1,180 @@
+var request= require('superagent');
+//importing the global variables
+var servicesConfig=require('../config/endPoints.json');
```
Comment (AlejandroJeanCarlo) [inline lib/servicesAPI.js]: you should use GLOBAL.initialDirectory

File: lib/servicesAPI.js [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+	});
+}
+exports.putRoom=putRoom;
+
+
+
+
+
+
+
```
Comment (AlejandroJeanCarlo) [inline lib/servicesAPI.js]: All this file should be deleted

File: tests/Smoke/services.js [hunk 4]
```diff
@@ -0,0 +1,156 @@
+//Services
+//smoke test from Joaquin Gonzales Mosquera
+var config = require('../../config/config.json');
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/services.js]: update this to use GLOBAL.initialDirectory+/config/config.json

File: tests/Smoke/services.js [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+var serviceConfig=require('../../config/service.json');
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/services.js]: same but use config.path.service

File: tests/Smoke/services.js [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+			});
+	});
+	after('After to create tests',function(done)
+	{
+		token = null;
+		done();
+	});
+	it('POST /services Smoke Test, Verify the status 200 after to add a new service',function(done)
+	{
+		servicesAPI
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/services.js]: re-factor to use roomManagerAPI

File: tests/Smoke/services.js [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+			.postServices(adminJson,token,function(err,resp)
+			{
+				expect(resp.status).to.equal(200);
+				done();
+			});
+	});
+	it('GET /servicesType SomkeTest, Verify the status 200',function(done)
+	{
+		servicesAPI
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/services.js]: same

File: tests/Smoke/services.js [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+			.getServiceType(function(err,res)
+			{
+				expect(res.status).to.equal(200);
+				done();
+			});
+	});
+	it('Get /services SmokeTest, Verify the status 200',function(done)
+	{
+		servicesAPI
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/services.js]: same

File: tests/Smoke/services.js [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+			.getServices(token,function(err,res)
+			{
+				expect(res.status).to.equal(200);
+				done();
+			});				
+	});
+	
+	it('GET /services/ServiceID Smoke test, Verify the status 200 (GET method) by serviceID',function(done)
+	{
+		servicesAPI
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/services.js]: to all test change to roomManagerAPI

File: tests/Smoke/services.js [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+			.getServices(token,function(err,resp){
+			idService=resp.body[0]._id;		
+			servicesAPI
+				.getServiceByID(idService,function(err,res)
+				{
+					expect(res.status).to.equal(200);
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/services.js]: try to use config.httpStatus.Ok in all the TC

File: tests/Smoke/services.js [hunk 11]
```diff
@@ -0,0 +1,156 @@
...
+								expect(re.status).to.equal(200);
+								done();
+							});
+					});
+			});
+	});
+	it('PUT /services/serviceID/rooms/roomID Smoke test, verify that it is possible modify a room with method PUT',function(done)
+	{
+		servicesAPI
+			.getServices(token,function(err,resp)
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/services.js]: you can reduce this using a before

File: lib/roomManagerAPI.js [hunk 12]
```diff
@@ -73,6 +73,83 @@ var getwithToken = function(token, endPoint, callback){
 		callback(err, res);
 		});
 };
+exports.getwithToken = getwithToken;//RoomManagerAPI.js
```
Comment (AlejandroJeanCarlo) [inline lib/roomManagerAPI.js]: You are duplicating code please review all the class

File: lib/servicesAPI.js [hunk 13]
```diff
@@ -0,0 +1,180 @@
+var request= require('superagent');
```
Comment (AlejandroJeanCarlo) [inline lib/servicesAPI.js]: All the class should be removed right?

File: tests/CRUD/services.js [hunk 14]
```diff
@@ -0,0 +1,88 @@
...
+var url = config.url;
+var serviceEndPoint = url+endPoints.services;
+var serviceEndPointPost=serviceEndPoint + serviceConfig.postFilter;;
+var serviceTypes = url+endPoints.serviceTypes;
+//Global Variables
+var token = null; 
+var idService = null;
+var idRoom = null;
+var roomEndPoint = serviceEndPoint;
+var rooms = endPoints.rooms;
```
Comment (AlejandroJeanCarlo) [inline tests/CRUD/services.js]: The name of those vars should be mode describing

File: tests/CRUD/services.js [hunk 15]
```diff
@@ -0,0 +1,88 @@
...
+					{
+						expect(res.status).to.equal(200);
+						expect(res.body[0]).to.have.property('type');
+						expect(res.body[0].type).to.equal(servicefromDB.type);
+						expect(res.body[0]).to.have.property('name');
+						expect(res.body[0].name).to.equal(servicefromDB.name);
+						expect(res.body[0]).to.have.property('version');
+						expect(res.body[0].version).to.equal(servicefromDB.version);
+						expect(res.body[0]).to.have.property('_id');
+						//assert.typeOf(servicefromDB._id, 'string');
```
Comment (AlejandroJeanCarlo) [inline tests/CRUD/services.js]: Remove this line please

File: lib/servicesAPI.js (hunk 13 above)
Comment (JoaquinGonzales) [inline lib/servicesAPI.js]: yes I will remove this file

**Pull request merged**